CONTRIBUTING.md

```diff
@@ -1,18 +1,17 @@
-﻿# Filing an issue
+# Filing an issue
 
 ### Must read
 * If you aren't sure, you can ask on the [**forum**](http://forum.qbittorrent.org) or read our [**wiki**](http://wiki.qbittorrent.org) first.
 * Do a quick **search**. Others might already reported the issue.
 * Write in **English**!
 * Provide **version** information: (You can find version numbers at menu `Help -> About -> Libraries`)
-  ```
+```
 qBittorrent:
```

Comment:
to ensure correct indents, 
add 2 spaces in the head of each line in this code block:
(2 spaces) 3x ticks
(2 spaces) qBittorrent:
(2 spaces) ...
(2 spaces) 3x ticks